### PR TITLE
PAY-2497:Changes to get secrets from new key vaults

### PR DESCRIPTION
**Pull request by amitchidrewar1301**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/fees-register-api/values.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 java:
-  image: 'hmctspublic.azurecr.io/fees-register/api:latest'
```
Comment (timja) [inline charts/fees-register-api/values.yaml]: revert

Comment (amitchidrewar1301) [reply charts/fees-register-api/values.yaml]: yep missed

**Comment by amitchidrewar1301** [comment]
> are you storing the app insights key in vault as well?

We need appinsights key for this app

File: infrastructure/main.tf [hunk 2]
```diff
@@ -22,9 +27,14 @@ data "azurerm_key_vault" "fees_key_vault" {
   resource_group_name = "${var.product}-${local.local_env}"
 }
 
+data "azurerm_key_vault" "payment_key_vault" {
+  name = "${local.core_product_vaultName}"
+  resource_group_name = "ccpay-${var.env}"
+}
+
 data "azurerm_key_vault_secret" "appinsights_instrumentation_key" {
```
Comment (timja) [inline infrastructure/main.tf]: you need to store the app insights key in your vault as well

File: api/src/main/resources/bootstrap.yaml [hunk 3]
```diff
@@ -3,7 +3,6 @@ spring:
     propertiesvolume:
       prefixed: false
       enabled: true
-      paths: /mnt/secrets/ccpay
+      paths: /mnt/secrets/fees-register
       aliases:
         api-POSTGRES-PASS: POSTGRES_PASSWORD
-        AppInsightsInstrumentationKey: azure.application-insights.instrumentation-key
```
Comment (timja) [inline api/src/main/resources/bootstrap.yaml]: why're you removing this? you still need it

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -13,39 +13,15 @@ def product = "fees-register"
...
-]
-
 def vaultOverrides = [
     'preview': 'aat',
     'hmctsdemo': 'aat',
 
 ]
 
-def secrets = [
-    'ccpay-${env}': [
```
Comment (timja) [inline Jenkinsfile_CNP]: don't you need these for your tests?

File: infrastructure/main.tf (hunk 2 above)
Comment (amitchidrewar1301) [reply infrastructure/main.tf]: its should be ok now I have added = which was missing

**Pull request merged**